community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tpl

```diff
@@ -22,16 +22,24 @@
     chown_owner: ${chown_owner}
     chgrp_group: ${chgrp_group}
   tasks:
+  - name: Create install directory
```

Comment:
Using `ansible.builtin.file` would be more correct as it would accurately reflect the changed property if the file system was modified.

```
- name: Create a directory if it does not exist
  ansible.builtin.file:
    path: /etc/some_directory
    state: directory
```